node/pegnet/snapshot.go

```diff
@@ -0,0 +1,156 @@
+package pegnet
+
+import (
+	"database/sql"
+	"fmt"
+
+	"github.com/Factom-Asset-Tokens/factom"
+	"github.com/pegnet/pegnetd/fat/fat2"
+)
+
+const (
+	// SnapshotRate is how often snapshots are taken and paid out on a block basis
+	SnapshotRate = 5 // Default is 144
```

Comment:
@sigrlami , why is it still 5 instead of 144?
I think that we should take a snapshot once per day.

Reply:
fixed to 144